src/Params/SUBnoteParameters.cpp

```diff
@@ -38,7 +38,7 @@ static const rtosc::Ports SUBnotePorts = {
     rSelf(SUBnoteParameters),
     rPaste,
     rToggle(Pstereo,    rShort("stereo"), rDefault(true), "Stereo Enable"),
-    rParamZyn(PVolume,  rShort("volume"), rDefault(96), "Volume"),
+    rParamF(Volume,  rShort("volume"), rDefault(0.47f), rLinear(-60.0f,20.0f), "Volume"),
```

Comment:
old eqn 0.1^(3*(1-Pvolume/96)). With an old default of 96 that means 0.1^(3*(1-96/96)) = 0.1^ (3*(1-1)) = 0.1^0 = 1. When a multiplier of one is converted to dB you have 20log_10(1) = +0dB.